src/isIsogram.test.js

```diff
@@ -6,4 +6,16 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
```

Comment:
Consider adding a test case to ensure that the `isIsogram` function is case-insensitive. For example, a test with input like 'Adam' should return `false` because 'A' and 'a' are considered the same letter according to the task requirements.